src/App.js

```diff
@@ -141,6 +168,59 @@ let App = () => {
           <a href="https://github.com/anucssa/anutimetable/issues">here</a>
       </Navbar.Text>
     </Navbar>
+
+    <div className={`fab ${settingsOpen ? 'fab--open' : ''}`} >
+      <Button
+        className={'fab-button'}
+        variant="secondary"
+        onClick={toggleSettings}
+      >
+        <svg className={settingsOpen ? 'hidden' : ''} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="32" height="32"><path fill="none" d="M0 0h24v24H0z"/><path d="M5.334 4.545a9.99 9.99 0 0 1 3.542-2.048A3.993 3.993 0 0 0 12 3.999a3.993 3.993 0 0 0 3.124-1.502 9.99 9.99 0 0 1 3.542 2.048 3.993 3.993 0 0 0 .262 3.454 3.993 3.993 0 0 0 2.863 1.955 10.043 10.043 0 0 1 0 4.09c-1.16.178-2.23.86-2.863 1.955a3.993 3.993 0 0 0-.262 3.455 9.99 9.99 0 0 1-3.542 2.047A3.993 3.993 0 0 0 12 20a3.993 3.993 0 0 0-3.124 1.502 9.99 9.99 0 0 1-3.542-2.047 3.993 3.993 0 0 0-.262-3.455 3.993 3.993 0 0 0-2.863-1.954 10.043 10.043 0 0 1 0-4.091 3.993 3.993 0 0 0 2.863-1.955 3.993 3.993 0 0 0 .262-3.454zM13.5 14.597a3 3 0 1 0-3-5.196 3 3 0 0 0 3 5.196z" fill="rgba(255,255,255,1)"/></svg>
+        <svg className={settingsOpen ? '' : 'hidden'} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="32" height="32"><path fill="none" d="M0 0h24v24H0z"/><path d="M12 10.586l4.95-4.95 1.414 1.414-4.95 4.95 4.95 4.95-1.414 1.414-4.95-4.95-4.95 4.95-1.414-1.414 4.95-4.95-4.95-4.95L7.05 5.636z" fill="rgba(255,255,255,1)"/></svg>
+      </Button>
+      <div className={'fab-actions'}>
+        <Button
+          className={'fab-action'}
+          variant={"primary"}
+          onClick={toggleStartDayDialog}
+        >
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24"><path fill="none" d="M0 0h24v24H0z"/><path d="M17 3h4a1 1 0 0 1 1 1v16a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V4a1 1 0 0 1 1-1h4V1h2v2h6V1h2v2zM4 9v10h16V9H4zm2 2h2v2H6v-2zm5 0h2v2h-2v-2zm5 0h2v2h-2v-2z" fill="rgba(255,255,255,1)"/></svg>
+        </Button>
+      </div>
+    </div>
+
+    <div className={`${startDayDialogOpen ? '' : 'hidden'} dialog-container`}>
+      <div className="dialog">
+        <Card>
+          <Card.Header>Calendar Start of Week</Card.Header>
+          <Card.Body>
+            <p>
+              The weekly calendar starts on&nbsp;
+              <select
```

Comment:
Would a [Dropdown](https://react-bootstrap-v4.netlify.app/components/dropdowns/) work here?

Reply:
I don't think so, as far as I could tell that component was for button dropdowns like you'd get in a menu, rather than a select input. I could theoretically use it but it'd add a lot of overhead and boilerplate over just using a basic HTML class.

Reply:
Also yeah it's probably possible to remove some of the css or refactor it into utlilty classes rather than semantic classes, I just figured for this it's so small that it's probably fine. I've removed a small amount now and I can probably run browser coverage over the rest at some point later.

Reply:
ahhh makes sense. the css comment was only cause Dropdowns had builtin bootstrap styling. they do have a [select](https://react-bootstrap.netlify.app/forms/select/) but I'm not worried, yours looks fine

Reply:
Ah, didn't see that 😅 
I think the bootstrap search might be broken for me 😂 
![image](https://user-images.githubusercontent.com/59380148/149619797-4ed5ed9a-5200-4f0f-acc8-945e7f6d6f11.png)

Reply:
yeah lol, had the same problem but I knew I'd seen it somewhere. pretty sure the Bootstrap 5 docs are better but FullCalendar has a hard dependency on 4 :(